rclpy/threading/data_plotter.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Yeah both could work, I would prefer being consistent across basic examples though.
I think that if you source/call `local_setup.bat` you can just run the executable that is on your path exactly like in linux, isn't it ?